Inventory.cpp

```diff
@@ -51,41 +51,42 @@ void Inventory::displayInventory() const
 	for (int i = 0; i < inventorySpace; i++)
 	{
 		std::cout << i + 1 << ". " << itemInSlot[i];
-		if (quantityInSlot[i] > 0){std::cout << " X" << quantityInSlot[i] << std::endl;}
-		else { std::cout << std::endl; }
+
```

Comment:
This is the common style for writing if-else pairs, specifically the placement of the braces.